runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -20,6 +20,8 @@
 
 #include <cker/Utils.h>
 #include <cker/train/operation/MaxPool.h>
+#include <cker/train/operation/AveragePool.h>
+#include <cker/operation/AveragePool.h>
```

Comment:
Could you reorder them like below?
```suggestion
#include <cker/operation/AveragePool.h>
#include <cker/train/operation/AveragePool.h>
#include <cker/train/operation/MaxPool.h>
```